src/anm/utils.js

```diff
@@ -199,6 +197,13 @@ function fit_rects(pw, ph, aw, ah) {
     } else return [ 1, [ 0, 0, aw, ah ] ];
 }
 
+function removeElement(array, element) {
```

Comment:
I thought [it's quite ok](http://bonsaiden.github.io/JavaScript-Garden/#core.delete) for [your own objects with no prototype chains](http://perfectionkills.com/understanding-delete/). Anyway, as noted above, some algorithms based on it, so please at least change it to `object[prop] = null`;